sphinx_rtd_theme/versions.html

```diff
@@ -1,4 +1,4 @@
-{% if READTHEDOCS %}
+{% if READTHEDOCS or display_lower_left %}
 {# Add rst-badge after rst-versions for small badge style. #}
   <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="versions">
     <span class="rst-current-version" data-toggle="rst-current-version">
```

Comment:
I think this should only be displayed on readthedocs

Reply:
@Blendify I'm not sure exactly what this comment means. The whole purpose of this PR is to add the ability to display the lower-left menu when not on readthedocs.

By default, it will not be displayed. If the user specifies `html_context['display_lower_left'] = True` in their sphinx onfig, then it is displayed. This is not an uncommon desired functionality. See:

 * https://stackoverflow.com/questions/49331914/enable-versions-in-sidebar-in-sphinx-read-the-docs-theme
 * https://stackoverflow.com/questions/49331914/enable-versions-in-sidebar-in-sphinx-read-the-docs-theme

This has been tested by the way, and you can see my changes downstream in-action at the following websites:

 * https://docs.buskill.in/buskill-app/en/stable/
 * https://vienneae.github.io/rtd-github-pages/en/master/index.html
 * https://nicozanf.github.io/py4web-doc/en/master/

This PR was created to push these changes back upstream so it would benefit the rest of the community.